src/main/docker/docker-compose.yml

```diff
@@ -437,13 +435,35 @@ services:
         LDAP_ADMIN_PASSWORD: "admin"
 
   ldap-seed:
-    image: osixia/openldap
+    image: ${LDAP_IMAGE}
     volumes:
       - ./ldap-seed:/container/service/slapd/assets/test/
     links:
       - ldap
-    command: chmod +x /container/service/slapd/assets/test/addAll.sh
-    entrypoint: sh -c '/container/service/slapd/assets/test/addAll.sh'
+    entrypoint: bash -c 'sleep 5s && ldapadd -h ldap -D "cn=admin,dc=example,dc=org" -w admin -f /container/service/slapd/assets/test/ldap-users-first.ldif'
+
+  ldap2:
```

Comment:
In order to test this in Travis, shouldn't these containers also be started in Travis integrationTest job then?

Reply:
I had missed this little detail... Done!